Header header 

uint32[] id                 # 表示此特征点ID
uint32[] cnt                # 表示此特征点被跟踪的次数

float32[] u                 # 此特征点畸变补偿后的像素坐标u
float32[] v                 # 此特征点畸变补偿后的像素坐标v

float32[] x                 # 此特征点畸变补偿后的归一化平面坐标x
float32[] y                 # 此特征点畸变补偿后的归一化平面坐标y
float32[] z                 # 此特征点畸变补偿后的归一化平面坐标z
